src/symon/index.ts

```diff
@@ -256,6 +265,17 @@ class SymonClient {
     this.reportProbesInterval = reportInterval ?? 10_000
 
     this.reportProbesLimit = reportLimit ?? 100
+
+    const { flags } = getContext()
+
+    this.isSymonExperimental = flags.symonExperimental
+
+    this.symonCouchDB =
+      flags.symonCouchDbURL || 'http://symon:symon@localhost:5984/symon'
+
+    this.pouch = new PouchDB('symon')
+
+    this.remotePouchDB = new PouchDB(this.symonCouchDB)
```

Comment:
the variable naming is a bit confusing. the flag is `symonCouchDBURL`, assigned to variable called `symonCouchDB`, then used to initiate a Pouch DB. 

Can you rename it so they make more sense?

Reply:
done renaming to symonCouchDB